src/client/components/componentWithSliders/componentWithSliders.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './componentWithSliders.css';
+
+import SelectItem from './selectItem';
+import AllSliders from './allSliders';
+import SaveButton from './saveButton';
+import UploadButton from './uploadButton';
+import Song from './song';
+
+const ComponentWithSliders = () => {
+  return (
```

Comment:
You can avoid return statement here, and simple use arrow functions return, func = () => ();

Reply:
fixed for all